website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,33 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
```

Comment:
Sorry to nitpick, but I don't think we need the "General" header here,  I know it's in the UI, but people aren't going to be searching for it and the stacked headers look silly as is:
```suggestion
```
I think it looks better without the extra header:
![Screenshot 2024-01-03 at 11 18 43 AM](https://github.com/hashicorp/terraform-docs-common/assets/32436232/62b46ac2-128a-4a99-ac03-613b8d26502e)